src/jobs/list-item.tsx

```diff
@@ -16,9 +24,16 @@ export default function JobListingItem({ item }: JobListingItemProps) {
     <section className={style.job__listing__container}>
       <div>
         <h3 className={and(style.job__title, 'fancy')}>{item.title}</h3>
-        <span>{item.location}</span>
+        <span>
+          {`${item.locations.map((location) => {
+            return ' ' + location.city;
+          })}`}
```

Comment:
```suggestion
          {item.locations.map((location) => {
            return ' ' + location.city;
          })}
```

Kunne man bare tatt bort disse? Evt med en `toString()` dersom jsx ikke gjør det for oss

Reply:
Av en eller annen uforklarlig grunn (for meg) sørger `${}`for å automatisk legge på komma mellom elementene. Det finnes sikkert en god grunn til hvorfor, men jeg tenkte at det var bedre å beholde de automagiske kommaene, enn å lage noe som ordna komma selv 🤔

Reply:
Det blir om til en array som gjøres toString når det template literals inn og da blir det automatisk komma. Om du vil gjøre det "korrekt" så kan det være slik:

```
          {item.locations.map((location) => {
            return location.city;
          }).join(', ')}
```

Reply:
Nå snakker vi, takk!

Reply:
Har oppdatert forslaget mitt med en toString() som ser ut til å gjøre så man får med seg kommaene. Kommer om forskjeller på hvordan jsx og toString()-funksjonen konkatinerer arrays når den skal stringifisere de. `${``}` er i praksis det samme som `toString()`, bare litt mindre eksplisitt.

Er ikke egentlig så viktig dette altså, men når jeg først kommenterte på det kan jeg like godt følge det opp 😅